docs/installation.md

```diff
@@ -1,189 +0,0 @@
----
```

Comment:
there should be a redirect if we are getting rid of this page

Reply:
done